File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.Arrays;
class Solution {
    public String solution(String s) {
        String [] arr = s.split(" ");
        
        int [] nums = new int[arr.length];
        for(int i = 0; i < nums.length; i++){
            nums[i] = Integer.valueOf(arr[i]);
        }
        Arrays.sort(nums);
        
        String max = String.valueOf(nums[nums.length-1]);
        String mini = String.valueOf(nums[0]);
        
        String answer = mini + " " + max; 
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cs
using System;
public class Solution {
    public string solution(string s) {
        string[] arr = s.Split(' ');
        int min = Int32.Parse(arr[0]);
        int max = Int32.Parse(arr[0]);

        for (int i = 1; i < arr.Length; i++) {
            int num = Int32.Parse(arr[i]);
            if (num < min) min = num;
            if (num > max) max = num;
        }

        return min.ToString() + " " + max.ToString();
    }
}
